src/main/java/com/dawn/jat/illuminati/comment/controller/CommentController.java

```diff
@@ -0,0 +1,4 @@
+package com.dawn.jat.illuminati.comment.controller;
+
+public class CommentController {
```

Comment:
this is just a suggestion, we dont need to add a Comment right now :d

Reply:
Removed

Reply:
Removed